qa/util/src/main/java/io/camunda/zeebe/qa/util/testcontainers/DefaultTestContainers.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.qa.util.testcontainers;
+
+import dasniko.testcontainers.keycloak.KeycloakContainer;
+import java.time.Duration;
+import org.testcontainers.utility.DockerImageName;
+
+public final class DefaultTestContainers {
```

Comment:
The idea is that for future containers which we reuse, we can share default config through this utility.